Code/IO/src/sitkImageViewer.cxx

```diff
@@ -0,0 +1,896 @@
+*  limitations under the License.
+*
+*=========================================================================*/
+#include "sitkImageViewer.h"
+#include "sitkMacro.h"
+#include "sitkImageFileWriter.h"
+#include <itkMacro.h>
+#include <itksys/SystemTools.hxx>
+#include <itksys/Process.h>
+#include <stdlib.h>
+#include <iostream>
+#include <sstream>
+#include <iterator>
+#include <string>
+#include <algorithm>
+#include <ctype.h>
+
+#ifdef _WIN32
+#include <process.h>
+#else
+#include <unistd.h>
+#endif
+
+
+//
+//  Dave's mental notes about ImageJ/Fiji and file formats
+//
+// ImageJ without plugins doesn't support MetaIO.
+// ImageJ displaying a color NifTi image by defaults displays R, G, and B as separate channels.
+// To display color NifTi images properly in ImageJ, you need to use the NIFTI_COLOR_MACRO
+//
+// Fiji does support MetaIO.
+// Fiji has the same color NifTi problem as ImageJ.
+//
+// Invoking Fiji is simpler then ImageJ because the command line flags are the
+// same on Windows and Linux.
+
+#define localDebugMacro(x)\
+  {                                                                     \
+    if (itk::simple::ImageViewer::GetDebug())                           \
+      {                                                                 \
+      std::ostringstream msg;                                           \
+      msg << "Debug: In " __FILE__ ", line " << __LINE__ << ": " x      \
+          << "\n\n";                                                    \
+      ::itk::OutputWindowDisplayDebugText( msg.str().c_str() );         \
+      }                                                                 \
+  }                                                                     \
+
+#define IMAGEJ_OPEN_MACRO "open(\"%f\"); rename(\"%t\");"
+#define NIFTI_COLOR_MACRO " run(\"Make Composite\", \"display=Composite\");"
+
+namespace itk
+{
+  namespace simple
+  {
+
+namespace
+{
+  // forward declaration of some helper functions
+  std::string ReplaceWords( const std::string command, const std::string app, const std::string filename,
+                            const std::string title, bool& fileFlag );
+  std::string UnquoteWord( const std::string word );
+  std::vector<std::string> ConvertCommand( const std::string command, const std::string app, const std::string filename,
+                                           const std::string title="" );
+  std::string FormatFileName ( const std::string TempDirectory, const std::string name, const std::string extension,
+                               const int tagID );
+  std::string BuildFullFileName( const std::string name, const std::string extension, const int tagID );
+#ifdef _WIN32
+  std::string DoubleBackslashes( const std::string word );
+#endif
+
+  void ExecuteCommand( const std::vector<std::string> & cmdLine, const unsigned int timeout=500 );
+}
+
+
+int ImageViewer::ViewerImageCount=0;
+bool ImageViewer::AreDefaultsInitialized=false;
```

Comment:
Change name to "GlobalDefaultsInitialized". The term "GlobalDefault" is used consistently through ITK/SimpleITK